src/client/modules/my/playerRankingTable/playerRankingTable.ts

```diff
@@ -1,34 +1,13 @@
 import { LightningElement } from 'lwc';
+import { getRawResults } from 'my/resultsFetcher';
```

Comment:
You can combine these into the same line:
`import { extractPlayerStats, getRawResults } from 'my/resultsFetcher';`